packages/data-fetcher/src/blockchain/blockchain.service.ts

```diff
@@ -180,14 +180,22 @@ export class BlockchainService implements OnModuleInit {
     return await erc20Contract.balanceOf(address, { blockTag });
   }
 
-  public async onModuleInit(): Promise<void> {
-    const bridgeAddresses = await this.getDefaultBridgeAddresses();
+  public async getBridgeContracts(): Promise<{
+    l1Erc20Bridge: string;
+    l1SharedDefaultBridge: string;
+    l2SharedDefaultBridge;
+  }> {
+    return await this.rpcCall(async () => {
+      return await this.provider.send("zks_getBridgeContracts", []);
+    }, "getBridgeContracts");
+  }
 
+  public async onModuleInit(): Promise<void> {
+    const bridgeAddresses = await this.getBridgeContracts();
     this.bridgeAddresses = {
-      l1Erc20DefaultBridge: bridgeAddresses.erc20L1.toLowerCase(),
-      l2Erc20DefaultBridge: bridgeAddresses.erc20L2.toLowerCase(),
+      l1Erc20DefaultBridge: bridgeAddresses.l1SharedDefaultBridge,
```

Comment:
How is it gonna work for the default Zk Sync network? There are no `l1SharedDefaultBridge` and `l2SharedDefaultBridge` in `zks_getBridgeContracts` response.
```
curl https://mainnet.era.zksync.io \
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"zks_getBridgeContracts","params":[ ],"id":1,"jsonrpc":"2.0"}'
```
returns:
```
{
  "jsonrpc": "2.0",
  "result": {
    "l1Erc20DefaultBridge": "0x57891966931eb4bb6fb81430e6ce0a03aabde063",
    "l1WethBridge": "0x0000000000000000000000000000000000000000",
    "l2Erc20DefaultBridge": "0x11f943b2c77b743ab90f4a0ae7d5a4e7fca3e102",
    "l2WethBridge": "0x0000000000000000000000000000000000000000"
  },
  "id": 1
}
```

Reply:
@SantiagoPittella we need to support the current schema with `l1Erc20DefaultBridge` and `l2Erc20DefaultBridge` as well as the new schema with `l1SharedDefaultBridge` and `l2SharedDefaultBridge`. This way we can deploy Explorer independently.

Reply:
Ok! Will do.

Reply:
I just updated it to support both cases.